src/test/python/complete_winml_tutorial.py

```diff
@@ -29,42 +29,42 @@ def sync_wrapper(*args, **kwds):
 
     return async_wrapper if asyncio.iscoroutinefunction(fun) else sync_wrapper
 
-import pyrt.windows.ai.machinelearning as winml 
+import winrt.windows.ai.machinelearning as winml 
```

Comment:
Why is the namespace lower case? Don't you honor the casing of the API?

Reply:
lowercase package names is python's convention. #201